test/powershell/Modules/Microsoft.PowerShell.Utility/Test-Json.Tests.ps1

```diff
@@ -103,4 +103,17 @@ Describe "Test-Json" -Tags "CI" {
         $errorVar[0].FullyQualifiedErrorId | Should -BeExactly "InvalidJsonAgainstSchema,Microsoft.PowerShell.Commands.TestJsonCommand"
         $errorVar[1].FullyQualifiedErrorId | Should -BeExactly "InvalidJsonAgainstSchema,Microsoft.PowerShell.Commands.TestJsonCommand"
     }
+
+    It "Test-Json recognizes primitives: <name>" -TestCases @(
+        @{ name = 'number'; value = 1 }
+        @{ name = '"true"'; value = '"true"' }
+        @{ name = '"false"'; value = '"false"' }
+        @{ name = '"null"'; value = '"null"' }
```

Comment:
Shouldn't these values be unquoted? Doesn't JSON accept `true`/`false`/`null` as primitive values?

Reply:
Perhaps you thing about ConvertTo-Json. Test-Json accepts string input.

Reply:
Sure, but that's already tested in the case below. Is this not intended to verify that it accepts the `true`/`false`/`null` literals?

Reply:
Ah, I see your point.
Add new tests